libraries/shared/factories/storage/keys_lookup_test.go

```diff
@@ -19,9 +19,9 @@ package storage_test
 import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/crypto"
-	"github.com/makerdao/vulcanizedb/libraries/shared/factories/storage"
+	storage_factory "github.com/makerdao/vulcanizedb/libraries/shared/factories/storage"
```

Comment:
Doesn't need to be addressed in this PR, but I'm not totally clear on how we distinguish what goes in `/factories/storage` vs `/storage` - and the fact that they're both useful for storage transformers probably makes this kind of aliasing necessary all over the place. Would be cool to reorganize stuff so that we only have one `storage` package likely to be imported in a given namespace

Reply:
Totally agree! I think this fits into the bigger restructuring of the project that we've mentioned - I'll add a story so we can set aside some time to do some deeper digging into how we want to structure things.

Reply:
new story: https://makerdao.atlassian.net/browse/VDB-1086